### PR TITLE
fix: typo squad

### PR DESCRIPTION
**Pull request by nensidosari**
## Changes

- squad instead of account

### Preview domain
https://fix-typo-squad-edit.preview.app.daily.dev

### CODE REVIEW

File: packages/shared/src/components/squads/settings/SquadDangerZone.tsx [hunk 1]
```diff
@@ -38,8 +38,8 @@ export function SquadDangerZone({ squad }: SquadDangerZoneProps): ReactElement {
       <DangerZone
         onClick={onDeleteSquad}
         className="mt-4"
-        cta="Delete account"
-        title="Deleting your account will:"
+        cta="Delete squad"
```
Comment (rebelchris) [inline packages/shared/src/components/squads/settings/SquadDangerZone.tsx:41]: Squad is a product term it's always capital S

```suggestion
        cta="Delete Squad"
```

File: packages/shared/src/components/squads/settings/SquadDangerZone.tsx [hunk 2]
```diff
@@ -38,8 +38,8 @@ export function SquadDangerZone({ squad }: SquadDangerZoneProps): ReactElement {
...
+        title="Deleting your squad will:"
```
Comment (rebelchris) [inline packages/shared/src/components/squads/settings/SquadDangerZone.tsx:42]:
```suggestion
        title="Deleting your Squad will:"
```

**Review by rebelchris — CHANGES_REQUESTED** [review]

File: packages/shared/src/components/squads/settings/SquadDangerZone.tsx (hunk 1 above)
Comment (nensidosari) [reply packages/shared/src/components/squads/settings/SquadDangerZone.tsx:41]: 😬

File: packages/shared/src/components/squads/settings/SquadDangerZone.tsx (hunk 2 above)
Comment (nensidosari) [reply packages/shared/src/components/squads/settings/SquadDangerZone.tsx:42]: In figma this is with lowercase S 🤔

Comment (rebelchris) [reply packages/shared/src/components/squads/settings/SquadDangerZone.tsx:42]: Yeah design does it all the time, you can look it up on slack there's been numerous chats.
Squad is a product term, therfor always capital :)

Comment (sshanzel) [reply packages/shared/src/components/squads/settings/SquadDangerZone.tsx:42]: Probably was just missed in design, but yeah, this should be on caps as discussed with product waaaaaay back.

Comment (rebelchris) [reply packages/shared/src/components/squads/settings/SquadDangerZone.tsx]: Just for reference:
https://dailydotdev.slack.com/archives/C05P9ET7S9K/p1699000372453029?thread_ts=1698997971.107649&cid=C05P9ET7S9K

**Review by rebelchris — APPROVED** [review]

**Pull request merged**